app/models/occupation_standard.rb

```diff
@@ -191,4 +191,12 @@ def show_national_occupational_framework_badge?
   def national?
     national_standard_type.present?
   end
+
+  def similar_programs_deprecated
```

Comment:
This method will need to be cleaned up after the feature is validated and the feature flag is turned on